flink-yarn/src/main/java/org/apache/flink/yarn/configuration/YarnConfigOptions.java

```diff
@@ -215,6 +215,14 @@
 				.noDefaultValue()
 				.withDescription("A semicolon-separated list of directories to be shipped to the YARN cluster.");
 
+	public static final ConfigOption<List<String>> SHARED_LIBRARIES =
+			key("yarn.shared-libraries")
+				.stringType()
+				.asList()
+				.noDefaultValue()
+				.withDescription("specify the path to use public visibility feature of " +
```

Comment:
Also, shall we capitalize the first letter? Like other config option description.

Reply:
Make sense. Will update.